### PR TITLE
docs: remove references to hambach testnet

### PR DESCRIPTION
**Pull request by dpdanpittman**
## Description

Closes: #1770

This PR removes any references to Hambach testnet.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: docs/ledger/get-started/README.md [hunk 1]
```diff
@@ -106,8 +102,6 @@ Additional packages and checksums are available under "Assets" on the [Release P
 
 The following installation instructions include two options, one for installing the `regen` binary with the "stable app configuration" and another with the "experimental app configuration".
```
Comment (ryanchristo) [inline docs/ledger/get-started/README.md:103]: This is not true anymore. Can you update?

File: docs/ledger/get-started/README.md [hunk 2]
```diff
@@ -142,12 +136,6 @@ Check out the latest stable version:
...
 
-*For the experimental app configuration (used on Hambach Testnet):*
-
-```bash
-git checkout v4.0.0
-```
-
 Build and install the `regen` binary:
 
 *For the stable app configuration (used on Regen Mainnet and Redwood Testnet):*
```
Comment (ryanchristo) [inline docs/ledger/get-started/README.md:141]: This doesn't seem necessary anymore. Can you remove throughout the docs?

**Review by ryanchristo — COMMENTED** [review]
Can you remove hambach information from the following page?

- https://deploy-preview-1779--regen-ledger-docs.netlify.app/ledger/

I don't think we need to separate mainnet and redwood instructions on the following page if they will always be the same:

- https://deploy-preview-1779--regen-ledger-docs.netlify.app/validators/get-started/install-regen.html

File: docs/validators/get-started/install-regen.md [hunk 3]
```diff
@@ -28,18 +28,6 @@ Check out the genesis version:
...
-
-*For Hambach Testnet:*
-
-```bash
-git checkout v4.0.0
-```
-
 Build and install the `regen` binary:
 
 *For Regen Mainnet:*
```
Comment (ryanchristo) [inline docs/validators/get-started/install-regen.md:33]: These are no longer needed if the instructions apply to all (i.e. mainnet and redwood)

**Comment by dpdanpittman** [comment]
Strange, my local repo has this already changed.  I pushed again

File: docs/ledger/get-started/README.md [hunk 4]
```diff
@@ -104,9 +100,7 @@ Additional packages and checksums are available under "Assets" on the [Release P
 
 ## Building From Source
 
-The following installation instructions include two options, one for installing the `regen` binary with the "stable app configuration" and another with the "experimental app configuration".
-
-If you are looking to interact with features on [Hambach Testnet](live-networks.md#hambach-testnet) not available on [Regen Mainnet](live-networks.md#regen-mainnet) and [Redwood Testnet](live-networks.md#redwood-testnet), then you will want to use the "experimental app configuration", which means you will need to add the `EXPERIMENTAL` option to the `make install` command.
+The following installation instructions includes one option for installing the `regen` binary with the "stable app configuration".
```
Comment (ryanchristo) [inline docs/ledger/get-started/README.md:103]:
```suggestion
The following installation instructions are for installing the `regen` binary.
```

File: docs/ledger/get-started/README.md [hunk 5]
```diff
@@ -170,14 +149,6 @@ regen version
...
-
-```bash
-v5.0.0
-```
-
-*For the experimental app configuration (used on Hambach Testnet):*
-
 ```bash
-v4.0.0
+v5.0.0
```
Comment (ryanchristo) [inline docs/ledger/get-started/README.md:153]:
```suggestion
v5.0.0
```

**Review by ryanchristo — APPROVED** [review]
LGTM

**Pull request merged**